site/documentation/content/api/command-and-control-pubsub/index.md

```diff
@@ -169,6 +204,9 @@ The semantics of the individual codes are specific to the device and command. Fo
 
 **Response Message sent from Hono Component**
 
+If the command response is an acknowledgement of a command by the Hono protocol adapter, it has a status code
+of *202* and the *content-type* attribute is set to *application/vnd.eclipse-hono-delivery-success-notification+json*.
```

Comment:
FMPOV this type of message should be added to the Event API's list of [well known events](https://eclipse.dev/hono/docs/api/event/#well-known-event-message-types).